test-data/unit/parse.test

```diff
@@ -2686,14 +2686,14 @@ MypyFile:1(
       OverloadedFuncDef:2(
         Decorator:2(
           Var(g)
-          NameExpr(foo)
+          NameExpr(overload)
           FuncDef:3(
             g
             Block:3(
               PassStmt:3())))
         Decorator:4(
           Var(g)
-          NameExpr(foo)
+          NameExpr(overload)
```

Comment:
They've stopped being interesting in the parse phase, but the semantic analysis phase files now have such tests.